OrbitGl/TimeGraph.cpp

```diff
@@ -42,7 +42,7 @@ TimeGraph::TimeGraph() : batcher_(BatcherId::kTimeGraph) {
       GetOrCreateThreadTrack(TracepointEventBuffer::kAllTracepointsFakeTid);
 
   // The process track is a special ThreadTrack of id "kAllThreadsFakeTid".
-  process_track_ = GetOrCreateThreadTrack(SamplingProfiler::kAllThreadsFakeTid);
+  process_track_ = nullptr;
```

Comment:
I though about this again, and that member is only used in the SortTracks method. So I would remove it from the class, and have it as a local variable in SortTracks

Reply:
Yes, you are right!